lfucg-exactions/server/templates/emails/email_template.html

```diff
@@ -0,0 +1,98 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html>
+
+<style>
+body {
+  margin: 15vh 10vw 20vh 10vw;
+  background-color: white;
+}
+
+#footer {
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 12vh;
+    background-color: rgb(0, 26, 73);
+    color: white;
+    text-align: right;
+    padding: 2vh 0;
+
+}
+
+#header {
+    position: fixed;
+    left: 0;
+    top: 0;
+    padding: 1% 10%;
+    width: 100%;
+    height: 10%;
+    background-color: rgb(0, 87, 168);
+    color: white;
+    display: flex;
+    line-height: 30px;
+    font-size: 3vh;
+    font-family: 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
+    letter-spacing: 3px;
+}
+
+#logo {
+    height: 50%;
+    padding: 30% 20px 30% 0px;
+}
+
+#rule {
+    width: 1px;
+    height:50%;
+    background-color:#FFFFFF;
+    margin: 18px 2vw;
+}
+
+#subheading {
+    font-weight: lighter;
+    letter-spacing: 6px;
+}
+
+#email-body {
+    background-color: white;
+    height: 100%;
+    text-align: center;
+}
+
+a {
+    color: white;
+}
+
+</style>
+
+
+<body>
+  <div id="header">
+    <a href="http://52.201.224.95/#/dashboard/">
+      <img id="logo" src="../../static/Lexington_Web_White_Horse_Only.png" alt="Lexington White Horse" href="http://52.201.224.95/#/dashboard/">
```

Comment:
With EX-78, you will be able to use StaticURL.  At a minimum, start with BASE_URL